blog/templates/post_details.html

```diff
@@ -12,7 +11,26 @@ <h1>{{ post.title }}</h1>
             <a href="{% url 'delete_post' post.pk %}">(Delete)</a>
         {% endif %}
     </small>
-    <hr>
-    {{ post.body|safe }}
+    <div class="post-detail-body">
+        {{ post.body|safe }}
+    </div>
+    <h2>Comments</h2>
+    {% if not post.comments.all %}
```

Comment:
Use 'for … empty' construction for show "No comments yet..." message

Reply:
done